externs/firebaseui-externs.js

```diff
@@ -231,7 +231,12 @@ firebaseui.auth.Config.prototype.signInFlow;
  * Determines the list of IdPs for handling federated sign-in, phone number as
  * well as password account sign-up.
  *
- * @type {!Array<!firebaseui.auth.SignInOption>|undefined}
+ * @type {!Array
```

Comment:
Shall we use the signInOption interface here as @bojeil-google suggested?

Reply:
Typescript doesn't allow extra properties by default...
https://stackoverflow.com/questions/33836671/typescript-interface-that-allows-other-properties

Reply:
Closure externs should allow it. It is ok if the 2 (closure externs and typescript definitions) don't align. The end result will be the same.